src/routes/(admin)/tests/test-[type]/[id]/question-selection/QuestionSelectionDialog.svelte

```
@@ -0,0 +1,238 @@
+	import { DataTable } from '$lib/components/data-table';
+	import { createQuestionSelectionColumns } from './columns';
+	import * as Dialog from '$lib/components/ui/dialog/index.js';
+	import Input from '$lib/components/ui/input/input.svelte';
+	import TagsSelection from '$lib/components/TagsSelection.svelte';
+	import StateSelection from '$lib/components/StateSelection.svelte';
+	import { goto } from '$app/navigation';
+	import { page } from '$app/state';
+	import type { QuestionForSelection } from './columns';
+	import type { ColumnDef } from '@tanstack/table-core';
+	import { Button } from '$lib/components/ui/button';
+	import CircleCheck from '@lucide/svelte/icons/circle-check';
```

ESLint

[error] 'CircleCheck' is defined but never used.

(@typescript-eslint/no-unused-vars)